Update Networking intro and move Multiplayer Gem topics underneath (#2051)
o3de/o3de.org: content/docs/user-guide/networking/multiplayer/running.md

---
linkTitle: Running Multiplayer Projects
title: Running Multiplayer Projects

> description: How to run multiplayer-enabled projects in Open 3D Engine.

Reviewer: ```suggestion
description: How to run multiplayer-enabled projects in Open 3D Engine (O3DE).
```

weight: 300
---

> Running projects using the Open 3D Engine Multiplayer Gem requires build steps followed by a variety of options to run the projects.

Reviewer: ```suggestion
Running projects using the **Multiplayer Gem** in **Open 3D Engine (O3DE)** requires build steps followed by a variety of options needed to run the projects.
```

* Ensure your project is [properly configured](./configuration).
* Build your project's `GameLauncher` and `ServerLauncher` targets.

> * Run the game either in standalone mode using the ImGui options to host, or via the in editor testing approach.

Reviewer: Are these prerequisites? A summary of the content of the topic? Add an intro sentence to the preceding paragraph to clarify for the reader.

Author: Rewrote and reorganized this intro.

## Testing in the Editor

> Multiplayer projects can be [run in Editor](./test-in-editor) using Ctrl-G. See the accompanying documentation for more detail.

Reviewer: ```suggestion
You can run multiplayer projects in **O3DE Editor** using **Ctrl+G**. For more information, refer to [Test Multiplayer Projects in the O3DE Editor](./test-in-editor).
```
